addons.html

```diff
@@ -103,6 +103,7 @@ <h3>Apps and Utilities:</h3>
                         href="https://github.com/AndreyPavlenko/transmissionbtc">transmissionbtc</a>,&nbsp; <a
                         href="https://github.com/6c65726f79/Transmissionic">Transmissionic</a>,&nbsp; <a
                         href="https://github.com/y-polek/TransmissionRemote">TransmissionRemote</a>,&nbsp; <a
+                        href="https://play.google.com/store/apps/details?id=eu.tmdpw.clutch">Clutch</a>,&nbsp; <a
```

Comment:
Could already be found using the link below. If there was a source code link for it then I'd be more than happy to add it, otherwise (and given that it says "In-app purchases" on that page), not sure why we'd want to advertise it.

@ckerr, thoughts?